src/instruments/src/EFB/Efb.tsx

```diff
@@ -72,6 +72,7 @@ class Efb extends React.Component<EfbProps, EfbState> {
         this.fetchSimbriefData = this.fetchSimbriefData.bind(this);
     }
 
+    // eslint-disable-next-line react/state-in-constructor
```

Comment:
Once again, this should just be moved to the constructor.

Reply:
Resolved.